Sources/SwiftDriver/Driver/Driver.swift

```diff
@@ -707,6 +706,17 @@ extension Driver {
   }
 }
 
+extension Diagnostic.Message {
+  public static func error_i_mode(_ driverKind: DriverKind) -> Diagnostic.Message {
+    .error(
+      """
+      the flag '-i' is no longer required and has been removed;
```

Comment:
Can you add a `\` at the end of this line, so it doesn't introduce a newline into the diagnostic text?
```suggestion
      the flag '-i' is no longer required and has been removed; \
```

Reply:
Of course, I thought that's what the original code was doing, but my C++ knowledge is fairly limited.